application/src/main/kotlin/net/corda/v5/application/crypto/HashingServiceUtils.kt

```diff
@@ -0,0 +1,47 @@
+@file:JvmName("HashingServiceUtils")
+
+package net.corda.v5.application.crypto
+
+import net.corda.v5.base.types.parseAsHex
+import net.corda.v5.crypto.DigestAlgorithmName
+import net.corda.v5.crypto.SecureHash
+
+/**
+ * Converts a digest represented as a {algorithm:}hexadecimal [String] into a [SecureHash].
+ *
+ * This function validates the length of the created digest. Prefer the usage of this function over [SecureHash.create].
+ *
+ * @param str The algorithm name followed by a delimiter and the sequence of hexadecimal digits that represents a digest.
+ *
+ * @throws IllegalArgumentException The input string does not contain the expected number of hexadecimal digits, or it contains
+ * incorrectly-encoded characters.
+ *
+ * @see SecureHash.create
+ */
+@Suppress("TooGenericExceptionCaught")
+fun HashingService.create(str: String): SecureHash {
```

Comment:
I think we generally refer to this as parse on other objects

Reply:
The name was left from C4 but anyway I've renamed it as well as SecureHash.create